lib/scripts.js

```diff
@@ -304,6 +304,7 @@ var scripts = {
    * Takes a lock
    */
   takeLock: function(queue, job, token, renew){
+    console.log('attempting lock for job id #', job.jobId)
     var lockCall;
```

Comment:
This log is removed in a later commit, I'll make sure I have no such console.* statements in the final PR.